friends/src/components/FriendsList.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import styled from 'styled-components'
+
+//styling the friend card for the list
+let Card = styled.div`
+  display: flex;
+`
+
+
+
+
+let Friends = (props) => {
+  return (
+  
+    <div className='list'>
+      <p>{props.friends.id}:</p>
```

Comment:
this is a good spot for object deconstruction.
```
const {id, name, age, email} = props.friends
```
```suggestion
      <p>{id}:</p>
```